lib/elements/element.py

```diff
@@ -274,6 +275,13 @@ def embroider(self, last_patch):
         return patches
 
     def fatal(self, message):
+        parser = ArgumentParser()
+        parser.add_argument("--extension")
+        my_args, remaining_args = parser.parse_known_args()
+
+        if my_args.extension == 'explain_validity':
+            return
```

Comment:
What is this for?  I assumed that `self.get_elements()` in `ExplainValidity` was throwing an error, and you had to do this to prevent it.  But when I tried removing this code, Explain Validity worked fine.  Maybe this is no longer necessary?